src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/MissingJavadocMethodCheck.java

```diff
@@ -112,13 +112,35 @@
  * &lt;module name="MissingJavadocMethod"/&gt;
  * </pre>
  * <p>
+ * Example:
+ * </p>
+ * <pre>
+ * public class Test {
+ *   public Test() {} // violation, missing javadoc for constructor
+ *   public void test() {} // violation, missing javadoc for method
+ *   &#47;**
+ *    * Some description here.
+ *    *&#47;
+ *   public void test2() {}
```

Comment:
default example should also show default values for properties, if applicable. Here `allowedAnnotations` and `scope` default values are missing

Reply:
Do you mean I need to include violations of allowed annotations and scope in this example?

Reply:
Done. I have made the changes according to my understanding. Please have a look and let me know if it's correct or not.